website/landingpage/public/blog/2020-06-18-welcome-to-luigi-blog.md

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
the first line I think shouldn't be empty, otherwise an error occurs

Reply:
Thank you, I deleted it